src/SignalR.Orleans/Groups/GroupMessage.cs

```diff
@@ -1,7 +0,0 @@
-namespace SignalR.Orleans.Groups
-{
-    public class GroupMessage
```

Comment:
The reason I was using this wrapper class, is due to serialization issues... Let's see how it goes...

Reply:
with this temporary fix `FallbackSerializationProvider = typeof(ILBasedSerializer).GetTypeInfo()` should be enough for the serializer issue. 
https://github.com/dotnet/orleans/issues/3640